content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -167,6 +169,276 @@ For each of them, the user must specify:
 
 The same Custom Local Disk can be used as source of cloning for all the VMs (same MachineDeployment or not) in the same cluster.
 
+---
+#### Scheduling settings
+It's possible to control how the tenant nodes are scheduled on the infrastructure nodes.
+![Scheduling](./Scheduling.png)
+
+We provide 3 different types of scheduling for the KubeVirt tenant nodes:
+- Ensure co-location on the same infrastructure node (*Pod Affinity Preset*).
+- Prevent co-location on the same infrastructure node (*Pod Anti Affinity Preset*).
+- Schedule on nodes having some specific labels (*Node Affinity Preset*).
+
+This setup is done in the `Initial Nodes` step of KKP dashboard when creating a cluster.
+
+![Dashboard](./Dashboard-scheduling.png) 
+
+
+For each of this scheduling types (*Pod Affinity Preset*, *Pod Anti Affinity Preset*, *Node Affinity Preset*), we can also specify if we want the affinity to be:
+- *"hard"* 
+-  *"soft"* 
+
+To achieve this goal, we use the [KubeVirt VM Affinity and Anti-affinity capabilities](https://kubevirt.io/user-guide/operations/node_assignment/#affinity-and-anti-affinity).
+
+
+*"hard"* or *"soft"* are related to the [Kubernetes: Assigning Pods to Nodes](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/). The following table shows the mapping:
+| Affinity type | Kubernetes affinity                             |
+|---------------|-------------------------------------------------|
+| hard          | requiredDuringSchedulingIgnoredDuringExecution  |
+| soft          | preferredDuringSchedulingIgnoredDuringExecution |
+
+
+#### How to set up the scheduling
+
+This set up is defined in the *MachineDeployment* `spec.affinity`:
+
+```yaml
+kind: MachineDeployment
+spec:
+ // ...
+  template:
+    spec:
+      providerSpec:
+        value:
+            // ....
+            affinity:
+              podAffinityPreset: "" # Allowed values: "", "soft", "hard"
+              podAntiAffinityPreset: "" # Allowed values: "", "soft", "hard"
+              nodeAffinityPreset:
+                type: "" # Allowed values: "", "soft", "hard"
+                key: "foo"
+                values:
+                  - bar
+// ....
+```
+
+**Important Note**:
+- `podAffinityPreset` and `podAntiAffinityPreset` are **mutually exclusive**. It does not make sense to schedule both on a single infrastructure node and prevent from being on the same infrastructure node.
+- `podAffinityPreset` can be specified along with `nodeAffinityPreset`: this allows ensure that the KubeVirt tenant nodes are co-located on a single infrastructure node that has some specific labels.
```

Comment:
I would change to
> on a single infrastructure node that has the specified labels (read how to add node labels)

We would have to add a link to part of the documentation that described how to specify labels in the UI.

Reply:
We can not label infrastructure nodes with the UI right ?

Reply:
you got me :) Nevermind then.